docs/SystemComponentsOverview.md

```diff
@@ -0,0 +1,104 @@
+- Import Analysis: Running a normal analysis for non-US cities doesn’t work because they don’t have census blocks or census-provided jobs data. However, it’s often possible to find equivalent data sources and run the analysis locally on a modified branch, and People For Bikes has been producing non-US analysis runs that way for some time. The “Import” functionality makes it possible to load and display the results of those customized analysis runs alongside the US ones produced by the normal process.
+
+**Deployment**
+Unlike the recent norm of deploying front ends by putting the static bundle on S3 and serving it via CloudFront, for this project the built asset bundle is copied into an Nginx container and served from an ECS service.
+
+## Back end
+
+The back end consists of a Django Rest Framework API and a PostgreSQL+PostGIS database.
+The main models are Neighborhood and AnalysisJob, and most of the other models are related to those two—either storing many-to-one relations or modeling parts of the workflow (e.g. AnalysisJobStatusUpdate). One exception is the Crash model, which stores crash data (from a static data source loaded via management command) for display on the map.
+
+One aspect of the models that’s worth noting is that the public site is organized around neighborhoods but most of the information that actually shows up there is on the AnalysisJob model, and while the Neighborhood instances are kept permanently and updated as needed (e.g. with changes to the parameters or border geometry), an AnalysisJob instance represents a single analysis run. So the scores, map layers, and downloadable results files on the neighborhood view come from the most recently-completed analysis run, and when a new run is completed, the map tile URLs and download links will change to include the new AnalysisJob instance’s UUID. Because almost every API call for neighborhood information requires at least some fields from the latest analysis, we added a denormalized field to the Neighborhood model, called `last_job`. It’s a nullable foreign key that points to the most recent successful analysis job for the neighborhood. The value is kept in sync by the `AnalysisJob.update_status()` method.
+
+### Django Q
+
+In addition to the Django service, we also run a [Django Q](https://django-q.readthedocs.io/en/latest/) service to handle some of the processing tasks that are triggered by API interactions but won’t fit in a request/response cycle. Namely:
+
+- Processing batch files, by opening them up, saving each geometry as a neighborhood, and submitting a Batch job for each.
+- Loading analysis imports, which includes opening the neighborhood_ways and neighborhood_census_blocks shapefiles and writing their contents to the database.
```

Comment:
Minor, but might be good to add quotation marks around neighborhood_ways and neighborhood_census_blocks - like what you did elsewhere in the doc

Reply:
I think I like it better as `neighborhood_ways` in these references to the shapefiles, but still "neighborhood_ways" in the first reference because there I'm drawing attention to it as a name, rather than a file, so quotes make more sense to me. Not wedded to it, though, so quotes everywhere or backticks everywhere would also be fine.